js/app/src/components/Dataset/ExampleComponents/Textbox.svelte

```diff
@@ -1,12 +1,33 @@
 <script lang="ts">
+	import { onMount } from "svelte";
 	import type { Value } from "../../Audio/types";
 
 	export let value: Value;
 	export let type: "gallery" | "table";
-	export let selected: boolean = false;
+	export let selected = false;
+
+	let size;
+	let el;
+
+	// $: el && size && set_styles(el, size);
+
+	function set_styles(element: HTMLElement, el_width: number): void {
+		if (!element || !el_width) return;
+		el.style.setProperty(
+			"--local-text-width",
+			`${el_width < 150 ? el_width : 200}px`
+		);
+		el.style.whiteSpace = "unset";
+	}
+
+	onMount(() => {
+		set_styles(el, size);
+	});
```

Comment:
We need to measure the element _without_ wrapping to see how wide it would be to decide what to do with it. We don't want to set a min width for anything that is smaller than 200px (which is arbitrary but a reasonable size) because then it would take up much more space than needed.